src/schema/estimates.rs

```diff
@@ -1,34 +0,0 @@
-//! Here we wrap some postgres functions to provide better row estimates for
-//! some postgres functions with default estimates which are off by and order
-//! of magnitude
-
-use pgx::*;
-
-// wrapper around jsonb_each_text to give a better row_estimate
-// for labels (10 not 100)
-extension_sql!(
-    r#"
-CREATE OR REPLACE FUNCTION @extschema@.label_jsonb_each_text(js jsonb, OUT key text, OUT value text)
-    RETURNS SETOF record
-    LANGUAGE internal
-    IMMUTABLE PARALLEL SAFE STRICT ROWS 10
-AS $function$jsonb_each_text$function$;
```

Comment:
The function definition in [base.sql](https://github.com/timescale/promscale_extension/blob/develop/migration/idempotent/001-base.sql#L641) differs from the definition here (that's language SQL this is language internal). That was done because you could only do language internal in extensions. But, language internal is much more efficient. So, we should replace the definition in base.sql with the definition from here as part of the same commit removing this.

Reply:
done